Classes/Emogrifier.php

```diff
@@ -537,20 +548,36 @@ private function generateStyleStringFromDeclarationsArrays(array $oldStyles, arr
         return trim($style);
     }
 
-
     /**
      * Copies the media part from CSS array parts to $xmlDocument.
      *
-     * @param string[] $cssParts
-     * @param \DOMDocument $xmlDocument
+     * @param string $mediaCss A string of CSS.
+     * @param \DOMDocument $xmlDocument The document to match against.
```

Comment:
Could you please lowercase the first letter of the @param descriptions and remove the period (as the descriptions are no full sentences)?